js/clinicalTimeline.js

```diff
@@ -261,6 +284,121 @@ window.clinicalTimeline = (function(){
   }
 
   /**
+   * Hanles the drawing and panning of the overviewAxis
+   */
+  function handleOverviewAxis() {
+    var tickTemp = getTickValues(minDays, maxDays, "months");
+    var minDayTick = tickTemp[0];
+    var maxDayTick =  tickTemp[tickTemp.length-1];
+    var zoomedWidth = chart.width();
+
+    var overview = d3.select(divId).append("svg")
+      .attr("height", 75)
+      .attr("width", overviewAxisWidth)
+      .attr("class", "overview");
+
+    var xScaleOverview = d3.time.scale()
```

Comment:
Try to add some comments to explain what `xScaleOverview`, `xScaleOverviewZoomed` and `xScaleRectangle` represent.